apps/frontend/src/api/news-and-events.ts

```diff
@@ -1,6 +1,13 @@
-import { ListNewsAndEventsResponse } from '@asap-hub/model';
+import {
+  ListNewsAndEventsResponse,
+  NewsOrEventResponse,
+} from '@asap-hub/model';
 
 import { useGetList } from './get-list';
+import { useGetOne } from './get-one';
 
 export const useNewsAndEvents = () =>
   useGetList<ListNewsAndEventsResponse>(`news-and-events`);
+
+export const useNewsOrEvent = (id: string) =>
```

Comment:
Should new api calls be done with recoil @jeysal?

Reply:
yeah, but didn't bring it up because I thought Filipe as a 1 hour per day frontend developer cannot be expected to stay on top of frontend trends all the time :D